selfdrive/debug/check_freq.py

```diff
@@ -43,7 +43,7 @@
     if t - prev_print > 1:
       print()
       for name in socket_names:
-        dts = np.diff(rcv_times[name])
+        dts = np.diff(rcv_times[name])  # type: ignore
```

Comment:
why is this needed?

Reply:
I haven't figured out why, but now mypy doesn't think `Deque[float]` is a `_SupportsArray`, but `MutableSequence` works fine